hack/config-default.sh

```diff
@@ -1,5 +1,5 @@
 binaries="cmd/virt-controller cmd/virt-launcher cmd/virt-handler cmd/virtctl cmd/fake-qemu-process cmd/virt-api cmd/subresource-access-test cmd/example-hook-sidecar"
-docker_images="cmd/virt-controller cmd/virt-launcher cmd/virt-handler cmd/virt-api images/disks-images-provider images/vm-killer cmd/container-disk-v1alpha images/cirros-container-disk-demo images/fedora-cloud-container-disk-demo images/alpine-container-disk-demo cmd/subresource-access-test images/winrmcli cmd/example-hook-sidecar images/cdi-http-import-server"
+docker_images="cmd/kubevirt-operator cmd/virt-controller cmd/virt-launcher cmd/virt-handler cmd/virt-api images/disks-images-provider images/vm-killer cmd/registry-disk-v1alpha images/cirros-registry-disk-demo images/fedora-cloud-registry-disk-demo images/alpine-registry-disk-demo cmd/subresource-access-test images/winrmcli cmd/example-hook-sidecar images/cdi-http-import-server"
```

Comment:
@irosenzw I think you by acciedent changed too much in this line (rebase merge conflict I guess): `s/registry/container/g`